source/features/github-bugs.css

```diff
@@ -62,8 +63,3 @@
 .js-issue-row .js-navigation-open ~ .text-small:last-child {
 	flex-wrap: wrap;
 }
-
-/* Restore margin between reactions on releases #5287 */
-[data-test-selector='release-card'] .social-reaction-summary-item:not(:first-child) {
-	margin-left: 8px !important;
-}
```

Comment:
Let's not drop this, it hasn't been fixed by GitHub yet:

![image](https://user-images.githubusercontent.com/46634000/153643137-855eec4c-46ea-4eb3-a6cd-7cc09a323c99.png)

Reply:
I removed this and it still worked, so I thought GitHub fixed it 😅 must be some build issues